src/functionalTest/java/uk/gov/hmcts/reform/translate/UniqueLoadTranslationEvaluator.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.translate;
+
+import uk.gov.hmcts.befta.player.BackEndFunctionalTestScenarioContext;
+
+import java.util.Map;
+
+public class UniqueLoadTranslationEvaluator implements CustomValueEvaluator {
+
+    @Override
+    public Boolean matches(CustomValueKey key) {
+        return CustomValueKey.UNIQUE_LOAD_TRANSLATION.equals(key);
+    }
+
+    @Override
+    public Object calculate(BackEndFunctionalTestScenarioContext scenarioContext, Object key) {
+        String englishPhrase = StringGenerator.generate();
+        return Map.of(englishPhrase, "");
```

Comment:
`return Map.of(StringGenerator.generate(), "");`

Reply:
done